roadmap.md

```diff
@@ -0,0 +1,59 @@
+(roadmap)=
+# Roadmap
+
+Last updated: February 2023
+
+## Our philosophy
+As stated in our homepage:
+
+> ArviZ is an open source project aiming to provide tools for Exploratory Analysis of Bayesian Models that do not depend on the inference library used.
+
+In this page we describe our current roadmap and specific actions to work towards this goal.
+
+## Project roadmap
+We believe both ArviZ and Exploratory Analysis of Bayesian Models are currently relatively established concepts.
+Therefore, generally speaking, as a project we are working on changing ArviZ paradigm/design
+from single monolithical functions that generate complete analysis and visualizations towards
+being more modular.
+
+At the same time, we continue to keep growing our team and publishing documentation as tasks at the same priority level as development.
```

Comment:
How about:
```
At the same time, we plan to equally prioritize all three tasks of:
- Library development
- Publishing documentation
- Growing and encouraging participation in the library
```

Reply:
Also, I am interested in what the third point means! Do we succeed by having more contributors to library development + more contributors to documentation, or do we also count talks/workshops/github discussions as "growing our team"?

Reply:
I think we need some more people across the board, but wouldn't put the emphasis there. Instead I would aim to ensure we have happy and engaged contributors, hopefully because their contributions help the library but also by being a part of ArviZ they manage to fulfill their own goals (whatever those are). I would definitely count all contributions, writing about ArviZ on a blog and adding it to [our community resources](https://python.arviz.org/en/stable/community.html#blogs-and-example-compilations), helping with twitter and mastodon, sprints/workshops/forum participation...